src/app/todo/services/task.service.ts

```diff
@@ -5,10 +5,30 @@ import { tasks, users } from './mockData';
 @Injectable({
   providedIn: 'root'
 })
-export class TaskService {
+export class TaskService {//По-любому с айдишкой некрасиво сделал, как было бы лучше?
+
+  private randomId: number;
+  private lastId: number;
 
   constructor() { }
 
+  private checkRepeated(): number {
+    this.randomId = this.getRandomIntNumber(1,100);
+    for(let task of tasks) {
+      if(Number(task.id) === this.randomId){
+        this.checkRepeated();
+      } else {
+        return this.randomId;
+      };
+    }
+    return this.randomId;
+  }
+
+  private getRandomIntNumber(min: number, max: number) {
+    min = Math.ceil(min);
```

Comment:
redefining function parameters is not really a good practice. It will be better if you create two new variables